opal/corelib/enumerable.rb

```diff
@@ -177,6 +177,10 @@ def count(object = undefined, &block)
   end
 
   def cycle(n = nil, &block)
+    if `arguments.length > 1`
```

Comment:
@wied03 The arity check for less than zero because the minimum arity is zero, the real problem is that a max-arity check is missing. Fixing it on master.